src/NuGet.Core/NuGet.Configuration/Settings/Items/AuthorItem.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using NuGet.Shared;
+
+namespace NuGet.Configuration
+{
+    public sealed class AuthorItem : TrustedSignerItem, IEquatable<AuthorItem>
+    {
+        public override string ElementName => ConfigurationConstants.Author;
+
+        protected override HashSet<string> RequiredAttributes => new HashSet<string>() { ConfigurationConstants.NameAttribute };
+
+        public AuthorItem(string name, params CertificateItem[] certificates)
+            : base(name, certificates)
+        {
+        }
+
+        internal AuthorItem(XElement element, SettingsFile origin)
+            : base(element, origin)
+        {
+        }
+
+        internal override SettingBase Clone()
+        {
+            var newItem = new AuthorItem(Name, Certificates.ToArray());
+
+            if (Origin != null)
+            {
+                newItem.SetOrigin(Origin);
+            }
+
+            return newItem;
+        }
+
+        internal override XNode AsXNode()
+        {
+            if (Node != null && Node is XElement)
+            {
+                return Node;
+            }
+
+            var element = new XElement(ElementName);
+
+            foreach (var cert in Certificates)
+            {
+                element.Add(cert.AsXNode());
+            }
+
+            foreach (var attr in Attributes)
+            {
+                element.SetAttributeValue(attr.Key, attr.Value);
+            }
+
+            return element;
+        }
+
+        public bool Equals(AuthorItem other)
+        {
+            if (other == null || other.GetType() != GetType())
+            {
+                return false;
+            }
+
+            if (ReferenceEquals(this, other))
+            {
+                return true;
+            }
+
+            return string.Equals(Name, other.Name, StringComparison.Ordinal);
+        }
+
+        public bool DeepEquals(AuthorItem other)
```

Comment:
Should these be internal/private if you're just gonna invoke them through the overrides?

Reply:
it could be made private, but it is this way for all other `SettingItems` do you think we should update this in all of them?

Reply:
I'd say unless you have a better reason not to. 

It doesn't have to be this PR. 
It can be a side PR that we can quickly go through.